JitsiPartyTeam/jitsi-party
# Feature/audio parameters store

Make parameter of panner node observable.

**How to test**
In `services\conference\src\scripts\stores\AudioParameters\index.ts`, add store to window variable.
Then in browser, change parameters manually to see the change of audio.
![image](https://user-images.githubusercontent.com/17405914/88761411-0b276c80-d1aa-11ea-9c95-8a35487306fc.png)
